test/letters/components/AddressContent.unit.spec.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import AddressContent from '../../../src/js/letters/components/AddressContent';
+
+const addressLines = {
+  streetAddress: '57 Columbus Strassa, Ben Franklin Village',
+  cityStatePostal: 'APO, AE 09028',
+  country: 'USA',
+};
+
+const defaultProps = {
+  saveError: false,
+  addressObject: addressLines,
+  name: 'Gary Todd',
+};
+
+describe('<AddressContent/>', () => {
+  it('should render', () => {
+    const tree = SkinDeep.shallowRender(<AddressContent { ...defaultProps }/>);
+    const vdom = tree.getRenderOutput();
+    expect(vdom).to.exist;
```

Comment:
Same as the comment above with all these `.to.exist` checks.

Reply:
👍